scripts/test

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -e
+
+function usage() {
+    echo -n \
+    "Usage: $(basename "$0")
+    Run tests and linter."
+}
+
+if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
+    if [ "${1:-}" = "--help" ]; then
+        usage
+    else
+        cd /vagrant
+
+        echo -e '\nRunning tests...'
+        npm run test
+
+        echo -e '\nRunning linter...'
+
+        # run linter with console output
+        npm run lint
```

Comment:
This is run twice intentionally on Jenkins, so the console output is available for reading without having to run it locally.